util/fetch_document_format_codelist.py

```diff
@@ -0,0 +1,59 @@
+"""
+Updates the document format codelist from the IANA media types CSV files.
+"""
+
+import csv
+import os
+import requests
+
+from helper import schema_dir
+
+
+def get_and_parse_csv(url):
```

Comment:
This method can be in-lined, since it is called only at one location.

Reply:
Done in https://github.com/open-contracting/standard/pull/1308/commits/bf809b43536493308a45c3dba75c6d432ceaead4